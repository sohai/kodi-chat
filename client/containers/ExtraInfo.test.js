/* global shallow describe it expect */
/* eslint-disable import/first */
import React from 'react';

import createStore from '../store';

import {
  ExtraInfoRender,
  mapStateToProps
} from './ExtraInfo';

const store = createStore();

describe('(Container) ExtraInfo', () => {
  const state = store.getState();
  const initialProps = mapStateToProps(state);
  const props = {
    ...initialProps
  };

  it('will receive right props', () => {
    expect(initialProps).toMatchSnapshot();
  });

  it('should render', () => {
    const wrapper = shallow(<ExtraInfoRender {...props} />, {
      context: { store }
    });
    expect(wrapper).toMatchSnapshot();
  });
});
