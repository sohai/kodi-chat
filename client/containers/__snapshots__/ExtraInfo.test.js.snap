// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Container) ExtraInfo should render 1`] = `<ExtraInfoComponent />`;

exports[`(Container) ExtraInfo will receive right props 1`] = `
Object {
  "contact": "",
  "typing": false,
}
`;
